config/initializers/sorcery.rb

```diff
@@ -0,0 +1,463 @@
+# The first thing you need to configure is which modules you need in your app.
+# The default is nothing which will include only core features (password encryption, login/logout).
+# Available submodules are: :user_activation, :http_basic_auth, :remember_me,
+# :reset_password, :session_timeout, :brute_force_protection, :activity_logging, :external
+Rails.application.config.sorcery.submodules = []
```

Comment:
Will include submodules later, as at the moment I can't understand what exactly we need.